techer.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(age:, specialization:, name: 'Unknown', parent_permission: true)
+    super(age: age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] To avoid any argument error, Kindly pass the parameter to the super-initialize in the same way they are defined in the Person class like:
```ruby
super(age, name, parent_permission: parent_permission)
```